glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
Hey, pkalever I am assuming you did not find any uses ofd dbus check_config too right?

I started working on the nl fix. I will post the nl fix patches in another PR when I am done.

Just finish any other changes requested in this patchset like the ones requested from lxbsz and ping me when you have repushed.

Go ahead and assume I am going to finish the nl patches around the same time you repush and just include the glfs check_config removal. I will handle the other ones in my PR.